transforms/universal/hap/python/Makefile

```diff
@@ -0,0 +1,62 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+include $(REPOROOT)/transforms/.make.transforms
+
+#TRANSFORM_RUNTIME_SRC_FILE=hap_transform_python.py
+TRANSFORM_NAME=hap
+
+include $(REPOROOT)/transforms/.make.transforms
+
+venv::	.transforms.python-venv
+
+install:: pip install -r requirements.txt
+
+test::	.transforms.python-test
+
+clean:: .transforms.clean
+
+image:: .transforms.python-image
+
+test-src:: .transforms.test-src
+
+setup:: .transforms.setup
+
+build:: build-dist image
+
+publish: publish-image
+
+publish-image:: .transforms.publish-image-python
+
+setup:: .transforms.setup
+
+# distribution versions is the same as image version.
+set-versions:
+	$(MAKE) TRANSFORM_PYTHON_VERSION=$(HAP_PYTHON_VERSION) TOML_VERSION=$(HAP_PYTHON_VERSION) .transforms.set-versions 
```

Comment:
These HAP_*VERSION require additions to .make.versions at the top of the repo.

Reply:
I will make the update in .make.versions but going forward, we should change that: it could be enough to define it in Makefile itself.

Reply:
fyi, at one point, maybe with kfp, these were required to be available more globally.  In addition, the ray/Makefile refers to the HAP_PYTHON_VERSION, so may not be able to change unless we coalesce runtimes into a single project.